src/Label.js

```diff
@@ -4,10 +4,10 @@ import theme, {colors} from './theme'
 import {COMMON} from './constants'
 
 const sizeMap = {
-  small: `padding 0.125em ${theme.space[1]}px; font-size: ${theme.fontSizes[0]}px;`,
-  medium: `padding: 3px ${theme.space[1]}px; font-size: ${theme.fontSizes[0]}px;`,
-  large: `padding: 4px ${theme.space[2]}px; ${theme.fontSizes[1]}px;`,
-  xl: `padding: ${theme.space[1]}px; ${theme.space[2]}px; ${theme.fontSizes[2]}px;`
+  small: `padding 0.125em ${theme.space[1]}; font-size: ${theme.fontSizes[0]}px;`,
```

Comment:
This is all removed in the base branch so we can probably omit this change
https://github.com/primer/components/pull/572/files#diff-1cfad437d41780bf359f56a0abe4e736R17